data/beta/field/public-body-account-classification.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: public-body-account-classification
+phase: beta
+register: public-body-account-classification
+text: A national accounting classification that a public body can have.
```

Comment:
can we come up with a better description please, maybe "that a public body can be assigned" or "that a public body can hold."

doesnt read right atm